source/settings/settingsproxy.cpp

```diff
@@ -1,5 +1,22 @@
 #include "settingsproxy.h"
 
+static QString encrypt(const QString& input) {
```

Comment:
As I've commented on `SettingsPage`, base64 doesn't really do anything in terms of secrecy. It's an encoding, not an encryption.